fsdb/Fsdb.py

```diff
@@ -49,7 +56,7 @@ def __init__(self, fsdbRoot, mode=None, deep=None, hash_alg=None):
             raise Exception("fsdb can not operate on relative path")
 
         # on different platforms same unicode string could have different rappresentation
-        if isinstance(fsdbRoot, unicode):
+        if isinstance(fsdbRoot, str):
```

Comment:
``` if isinstance(fsdbRoot, str):``` this works only on python3

On [python2](https://docs.python.org/2/howto/unicode.html#python-2-x-s-unicode-support) str and unicode are different class